dev-python/sphinx/files/sphinx-1.4.1-docutils13.patch

```diff
@@ -0,0 +1,34 @@
+# git diff 3adc236114^..7dabcdee91 -- sphinx/writers/html.py
+# and parts of edfdd3ec78 as well in order to make these apply
+
+diff --git a/sphinx/writers/html.py b/sphinx/writers/html.py
+index 7b666aa5..d6bdd833 100644
+--- a/sphinx/writers/html.py
++++ b/sphinx/writers/html.py
+@@ -458,9 +458,9 @@ class HTMLTranslator(BaseTranslator):
+             node['uri'] = posixpath.join(self.builder.imgpath,
+                                          self.builder.images[olduri])
+ 
+-        if node['uri'].lower().endswith('svg') or \
+-           node['uri'].lower().endswith('svgz'):
+-            atts = {'src': node['uri']}
++        uri = node['uri']
++        if uri.lower().endswith(('svg', 'svgz')):
++            atts = {'src': uri}
+             if 'width' in node:
+                 atts['width'] = node['width']
+             if 'height' in node:
+@@ -492,6 +492,13 @@ class HTMLTranslator(BaseTranslator):
+                         node['height'] = str(size[1])
+         BaseTranslator.visit_image(self, node)
+ 
++    # overwritten
++    def depart_image(self, node):
++        if node['uri'].lower().endswith(('svg', 'svgz')):
```

Comment:
Upstream reverted that version check in https://github.com/sphinx-doc/sphinx/commit/b98e2e39bb71795d1edc4c0696c3606a57d936ff. The important aspect is to keep things balanced: if `visit_image` does call the base version without modifying the stack itself, then so does `depart_image`. If `visit_image` returns without calling the base version, and does push something on the stack, then `depart_image` doesn't call the base version either, and makes use of the item on the stack. The common check for file extension achieves that these code paths will always match one another.